@xen-orchestra/web/src/components/pool/network/PoolHostInternalNetworksTable.vue

```diff
@@ -159,7 +114,7 @@ const filteredNetworks = computed(() => {
 
 const networkIds = computed(() => networks.map(network => network.id))
 
-const { selected, areAllSelected } = useMultiSelect(networkIds)
+const { selected } = useMultiSelect(networkIds)
 
 const toggleSelect = () => {
   selected.value = selected.value.length === 0 ? networkIds.value : []
```

Comment:
You can delete these lines and remember to remove `@toggle-select` in the template:
```suggestion
```